docs/docs/getting-started/v3-migration.md

```diff
@@ -279,6 +280,8 @@ The following properties and methods were removed:
 
 * `IPlugin.afterScaleUpdate`. Use `afterLayout` instead
 * `Legend.margins` is now private
+* Legend `onClick`, `onHover`, and `onLeave` options now receive the legend as the 3rd argument rather than implicitly via `this`
+* Legend `onClick`, `onHover`, and `onLeave` options now receive a wrapped `event` as the first parameter. `event.native` accesses the parameter from v2.
```

Comment:
It took me awhile to understand what "`event.native` accesses the parameter from v2." meant. How about "The previous first parameter value can now be accessed as `event.native`"?

Reply:
Good call. I've update